cranelift-codegen/src/ir/instructions.rs

```diff
@@ -652,6 +656,7 @@ mod tests {
             ints: BitSet8::from_range(4, 7),
             floats: BitSet8::from_range(0, 0),
             bools: BitSet8::from_range(3, 7),
+            refs: BitSet8::from_range(4, 7),
```

Comment:
Can you add assertions on what you expect below, next to the rest of the assertions?